src/test/java/com/jwplayer/southpaw/topic/ConsoleTopicTest.java

```diff
@@ -83,7 +83,7 @@ public void testReadNext() {
 
     @Test(expected = NotImplementedException.class)
     public void testResetCurrentOffset() {
```

Comment:
Nit, do we want to update the two test names in this file change to align with the plural form? `testResetCurrentOffsets`